interface/main/calendar/add_edit_event.php

```diff
@@ -1582,26 +1597,38 @@ function find_available(extra) {
     dateChanged();
 });
 
-// Check for errors when the form is submitted.
-function validate(valu) {
-     var f = document.getElementById('theform');
-    if (f.form_repeat.checked &&
-        (! f.form_enddate.value || f.form_enddate.value < f.form_date.value)) {
-        alert('<?php echo addslashes(xl("An end date later than the start date is required for repeated events!")); ?>');
-        return false;
+/*
+* validation on the form with new client side validation (using validate.js).
+* this enable to add new rules for this form in the pageValidation list.
+* */
+var collectvalidation = <?php echo($collectthis); ?>;
+function validateform(valu){
+    //add rule if choose repeating event
+    if ($('#form_repeat').is(':checked')){
+        collectvalidation.form_enddate = {
+            datetime: {
+                dateOnly: true,
+                earliest: $('#form_date').val(),
+                message: "An end date later than the start date is required for repeated events!"
```

Comment:
the translate is found with all translations in the validation_script.js.php line 227